resonance/cross/windowize_by_events.py

```diff
@@ -0,0 +1,151 @@
+import resonance.si as si
+import resonance.db as db
+from resonance.internal import declare_transformation, Processor
+import numpy as np
+import numpy_ringbuffer as np_rb
+import math
+
+@declare_transformation
+# Split data stream to windows using event stream
+# @param data Data stream
```

Comment:
Description does not match signature

In general signature is crappy, try to come with better names and proper descriptions for arguments